assets/layers/shower/shower.json

```diff
@@ -0,0 +1,184 @@
+{
+  "id": "shower",
+  "name": {
+    "en": "Shower"
+  },
+  "minzoom": 12,
+  "source": {
+    "osmTags": "amenity=shower"
+  },
+  "title": {
+    "render": {
+      "en": "Shower"
+    }
+  },
+  "presets": [
+    {
+      "title": {
+        "en": "a shower"
+      },
+      "description": {
+        "en": "A (public) shower"
+      },
+      "tags": [
+        "amenity=shower"
+      ]
+    }
+  ],
+  "tagRenderings": [
+    "images",
+    "level",
+    {
+      "id": "access",
+      "question": {
+        "en": "Who can use this shower?"
+      },
+      "mappings": [
+        {
+          "if": "access=yes",
+          "then": {
+            "en": "Anyone can use this shower"
+          }
+        },
+        {
+          "if": "access=customers",
+          "then": {
+            "en": "Only customers can use this shower"
+          }
+        },
+        {
+          "if": "access=key",
+          "then": {
+            "en": "Accesible, but one has to ask for a key"
+          }
+        }
+      ]
+    },
+    {
+      "id": "fee",
+      "question": {
+        "en": "Is there a fee for using this shower?"
+      },
+      "mappings": [
+        {
+          "if": "fee=yes",
+          "then": {
+            "en": "There is a fee for using this shower"
+          }
+        },
+        {
+          "if": "fee=no",
```

Comment:
Perhaps add an `addExtraTags: ["charge="]` here? This will remove the `charge`-tag again

Reply:
Good catch, added

Reply:
It's rare for me to spot a possible improvement in your PR's!